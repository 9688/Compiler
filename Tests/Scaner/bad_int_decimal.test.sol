Ошибка в строке 1 позиции 2: недопустимый символ "x" в записи числа.
IDENTIFICATOR 1 3 x1231
EOF 1 8 
