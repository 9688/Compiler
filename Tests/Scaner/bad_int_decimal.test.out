Ошибка в строке 1 позиции 2: недопустимый символ "x" в записи числа.
EOF 1 8 
